ProcessLib/RichardsMechanics/RichardsMechanicsFEM-impl.h

```diff
@@ -508,17 +533,18 @@ void RichardsMechanicsLocalAssembler<ShapeFunctionDisplacement,
         local_Jac
             .template block<pressure_size, pressure_size>(pressure_index,
                                                           pressure_index)
-            .noalias() += dNdx_p.transpose() * rho_LR * K_over_mu * grad_p_cap *
-                          dk_rel_dS_l * dS_L_dp_cap * N_p * w;
+            .noalias() += dNdx_p.transpose() * rho_LR / mu * K_intrinsic *
+                          grad_p_cap * dk_rel_dS_l * dS_L_dp_cap * N_p * w;
 
         local_Jac
             .template block<pressure_size, pressure_size>(pressure_index,
                                                           pressure_index)
-            .noalias() += dNdx_p.transpose() * rho_LR * rho_LR * K_over_mu * b *
-                          dk_rel_dS_l * dS_L_dp_cap * N_p * w;
+            .noalias() += dNdx_p.transpose() * rho_LR * rho_LR / mu *
```

Comment:
Sounds missing krel.

Reply:
I would think that here dk_rel/dS_l is used instead of k_rel. Same for the other derivative expression in line ~530.